components/root_layout/root_layout.vue

```diff
@@ -17,11 +17,17 @@
       :sidebar-width="sidebarWidth"
       :sidebar-position="sidebarPosition"
     >
-      <template #sidebar>
+      <template
+        v-if="$slots.sidebar"
```

Comment:
This doesn't seem to be an issue in the Vue2 implementation and works as expected. Since there's an internal change in how the slots are passed between Vue2 and Vue3, I think makes sense to leave the fix just for the vue3 implementation. What do you think?

Reply:
I think its better to keep both implementations as close as possible, I'd prefer to maintain this implementation here too.

Reply:
> I think its better to keep both implementations as close as possible

Yeah I totally agree with that but why add a fix for an issue we don't have in vue2?

Reply:
I mean, not a big deal adding this specific fix since there won't be any problem but wondering on applying the same criteria in more largar fixes.

Reply:
Yeah, you're right, we need to consider that too. I'll merge this but lets talk about this on our next grooming.